### PR TITLE
Add rpc wallet history

### PR DESCRIPTION
**Pull request by PawelPodkalicki**

### CODE REVIEW

File: app/Http/Controllers/Rpc/WalletController.php [hunk 1]
```diff
@@ -137,6 +140,38 @@ public function depositInfo()
         return self::json($resp);
     }
 
+    public function history()
+    {
+        $userId = Auth::user()->id;
+        $resp = [];
+        //TODO pagination
```
Comment (adshares-tech) [inline app/Http/Controllers/Rpc/WalletController.php]: INFO Complete the task associated to this "TODO" comment. [rule](https://sonarcloud.io/coding_rules#rule_key=php%3AS1135)

File: app/Http/Controllers/Rpc/WalletController.php [hunk 2]
```diff
@@ -137,6 +140,38 @@ public function depositInfo()
...
+        foreach (UserLedger::where('users_id', $userId)->cursor() as $ul) {
+            $amount = AdsConverter::clicksToAds($ul->amount);
+            $date = $ul->created_at->format(Carbon::RFC7231_FORMAT);
+            $txid = $ul->txid;
+            if (null !== $txid) {
+                // TODO export blockexplorer address / update testHistory in WalletControllerTest
```
Comment (adshares-tech) [inline app/Http/Controllers/Rpc/WalletController.php]: INFO Complete the task associated to this "TODO" comment. [rule](https://sonarcloud.io/coding_rules#rule_key=php%3AS1135)

File: app/Jobs/AdsSendOne.php [hunk 3]
```diff
@@ -81,7 +81,7 @@ public function handle(AdsClient $adsClient)
         if (AdsValidator::isTransactionIdValid($txid)) {
             // TODO move to queue (or service in general) to assure user ledger entry update
```
Comment (adshares-tech) [inline app/Jobs/AdsSendOne.php:82]: INFO Complete the task associated to this "TODO" comment. [rule](https://sonarcloud.io/coding_rules#rule_key=php%3AS1135)

**Comment by adshares-tech** [comment]
SonarQube analysis reported 3 issues
* ![INFO][INFO] 3 info

Watch the comments in this conversation to review them.

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'

File: app/Http/Controllers/Rpc/WalletController.php [hunk 4]
```diff
@@ -137,6 +140,38 @@ public function depositInfo()
...
+                $link = "https://operator1.e11.click/blockexplorer/transactions/${txid}";
```
Comment (c3zi) [inline app/Http/Controllers/Rpc/WalletController.php]: Please move Blockexplorer url to a dedicated variable in `.env`

File: app/Http/Controllers/Rpc/WalletController.php (hunk 1 above)
Comment (PawelPodkalicki) [reply app/Http/Controllers/Rpc/WalletController.php]: I will do it.

**Review by c3zi — CHANGES_REQUESTED** [review]

File: app/Http/Controllers/Rpc/WalletController.php (hunk 4 above)
Comment (PawelPodkalicki) [reply app/Http/Controllers/Rpc/WalletController.php]: OK, I will move it.

**Review by c3zi — APPROVED** [review]

**Pull request merged**